src/content.ts

```diff
@@ -143,3 +144,26 @@ config.getAsync("modeindicator").then(mode => {
         if (config.get("modeindicator") !== "true") statusIndicator.remove()
     })
 })
+
+// loading style toggles on page load
+config.getAsync("styletoggles").then(styletoggles => {
```

Comment:
I don't see the problem with DocStart:

String.search() allows you to use a regex, so you can easily match multiple URLs with pipes, e.g. `autocmd DocStart /www.(reddit|google).com/`. Do you just think this is less user friendly?

> to use autocmd on DocStart I would have to include a hidden load mode on excmd style

I think `styleapply [name]` is a totally reasonable thing to have. One could use it to roll one's own reader mode, for example.

Reply:
Using `DocStart` instead of this would also save you from having to use messages explicitly.

Reply:
Thanks for the review. I'm currently busy with some studio recording stuff but it will not take much longer.